### ISSUE TITLE
Video_Game_Sales 

### ISSUE THREAD

**Issue by erikk-avans** [hansvana/ML5_NN_Trainer#15]
Getting  while training the model "error in inputArray of normalizeArray() function"
Regression
Dataset 
[vgsales_v13.csv.zip](https://github.com/hansvana/ML5_NN_Trainer/files/4724086/vgsales_v13.csv.zip)

Input : Year, Genre, Platform
Output : Global Sales

**Comment by hansvana** [comment]
The problem is fixed in version 0.2.3, but this dataset has problems that need to be solved before training:
- The "year" feature has bad data, like 0 or NA
- There are some bad rows that would need to be manually filtered, like
```
"16567,Original Frisbee Disc Sports: Ultimate & Golf,DS,2007,Action,""Destination Software"""" Inc0.010000.01"""""",,,,,"
```

The size of the dataset (16572 items) causes noticeable slowdown on my pc too.

**Comment by erikk-avans** [comment]
Cleaned up and reduced the dataset. It is now on https://github.com/CMD-Breda-Avans/Datasets/blob/master/DATASETS.md
Problem : global sales shows same number for all predictions.
Input : Year, Genre, Platform
Output : Global Sales
